src/components/Layout.tsx

```diff
@@ -14,28 +14,26 @@ type Props = {
 const Layout = ({ text, icon, children }: Props) => {
   return (
     <>
-      <Flex alignItems="center" h={36} bg="teal.500">
+      <Flex bgColor="teal.500" h={36} align="center">
         <HSpacer size={16} />
-        <HStack w="100%" h="100%">
-          <HStack w="90%" alignItems="start">
-            <VSpacer size={1} />
-            <Icon as={icon} boxSize={6} color="teal.600" />
-            <Text
-              fontSize="lg"
-              lineHeight={7}
-              fontWeight="semibold"
-              color="white"
-              whiteSpace="pre-line"
-            >
-              {text}
-            </Text>
-          </HStack>
+        <Flex>
+          <Icon as={icon} boxSize={6} color="teal.600" />
+          <HSpacer size={1} />
+          <Text
+            fontSize="lg"
+            fontWeight="semibold"
+            color="white"
+            whiteSpace="pre-line"
+          >
+            {text}
+          </Text>
+        </Flex>
 
-          <Box h="100%" w="10%" align="right">
-            <Menu />
-          </Box>
-        </HStack>
-        <HSpacer size={3} />
+        <Spacer />
+
+        <Box h={10} w={10} alignSelf="flex-start">
```

Comment:
ここのサイズ指定は効いてないな
`Menu` コンポーネントがスペース (ハンバーガーアイコンの外側にパディング領域) を持ってそうなので、そっちのリファクタリングの際に必要に応じてこの `Box` もサイズ指定するのがいいかも

```suggestion
        <Box alignSelf="flex-start">
```

Reply:
ここスタイル調整でテスト用に書いてたのが残ってるわ🤣
消します